build.gradle

```diff
@@ -8,8 +8,8 @@ plugins {
     id "maven-publish"
     id "signing"
     id 'com.github.spotbugs' version '4.6.0'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
-    id 'org.jetbrains.kotlin.plugin.spring' version '1.3.72'
+    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jetbrains.kotlin.plugin.spring'

```

Comment:
The build is failing with

```
Plugin [id: 'org.jetbrains.kotlin.jvm'] was not found in any of the following sources:
```

Don't you need the `version`? Other projects like `rhizome` have it:

```
id 'org.jetbrains.kotlin.jvm' version '1.5.10'
id 'org.jetbrains.kotlin.plugin.spring' version '1.5.10'
```